--- smb.conf	2019-07-06 07:25:03.246649208 -0700
+++ /etc/samba/smb.conf	2019-07-06 07:27:11.144614312 -0700
@@ -26,7 +26,12 @@
 ## Browsing/Identification ###
 
 # Change this to the workgroup/NT-domain name your Samba server will part of
-   workgroup = WORKGROUP
+   workgroup = __SEDWORKGROUPNAMECAP__
+   client signing = yes
+   client use spnego = yes
+   kerberos method = secrets and keytab
+   realm = __SEDFQDNCAP__
+   security = ads
 
 # server string is the equivalent of the NT Description field
 	server string = %h server (Samba, Ubuntu)

